src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -89,6 +89,18 @@
 				{{ t('spreed', 'Sounds for chat and call notifications can be adjusted in the personal settings.') }} ↗
 			</a>
 		</NcAppSettingsSection>
+		<NcAppSettingsSection id="calls"
+			:name="t('spreed', 'Calls')"
+			class="app-settings-section">
+			<NcCheckboxRadioSwitch id="call-media"
```

Comment:
Move it up, close to devices, or join into a single section?

Reply:
> close to devices, or join into a single section?

I'd say so, that's where I'd expect to find it

Reply:
Put it right under video preview in the same section or keep it separate?

Reply:
same section sounds good enough